format/FlightSql.proto

```diff
@@ -867,6 +867,45 @@ enum SqlSupportsConvert {
   SQL_CONVERT_VARCHAR = 19;
 }
 
+/*
+ * Represents a request to retrieve information about data type supported ona Flight SQL enabled backend.
+ * Used in the command member of FlightDescriptor for the following RPC calls:
+ *  - GetSchema: return the schema of the query.
+ *  - GetFlightInfo: execute the catalog metadata request.
+ *
+ * The returned schema will be:
+ * <
+ *   type_name: utf8 not null,
+ *   data_type: int not null,
+ *   column_size: int,
+ *   literal_prefix: utf8,
+ *   literal_suffix: utf8,
+ *   create_params: utf8
+ *   nullable: int not null,
+ *   case_sensitive: bool null,
+ *   searchable: int not null,
+ *   unsigned_attribute: bool,
+ *   fixed_prec_scale: bool null,
+ *   auto_increment: bool,
+ *   local_type_name: utf8,
+ *   minimum_scale: int,
+ *   maximum_scale: int,
+ *   sql_data_type: int not null,
+ *   sql_datetime_sub: int,
+ *   num_prec_radix: int,
+ *   interval_precision. int
```

Comment:
Can we comment on what each of these fields is meant to be? I suppose they map 1:1 with JDBC, but we should have descriptions inline.

What is (if any) the ODBC equivalent? Does this cover that?

Reply:
> What is (if any) the ODBC equivalent? Does this cover that?

Yes, they cover it. Almost all them have identical names, just a few have a small difference in their names.

e.g.
` *   column_size: int,`
The value above in the JDBC is called "PRECISION" and in the ODBC "COLUMN_SIZE"

Reply:
Added the comments.

> What is (if any) the ODBC equivalent? Does this cover that?

Short answer, yes, it does. All the values were thought thinking on the integration between JDBC and ODBC